sdl_android/src/main/java/com/smartdevicelink/util/Version.java

```diff
@@ -40,6 +40,24 @@ public int getPatch() {
         return patch;
     }
 
+    /**
+     * Method to test if this instance of Version is newer than the supplied one.
+     * @param version the version to check against
+     * @return 1 if this instance is newer, -1 if supplied version is newer, and 0 if they are equal
+     */
+    public int isNewerThan(Version version){
```

Comment:
I think there is an issue with the conditions here. According to the logic, 6.0.0 will be equal to 5.0.0.

Reply:
Agreed

Reply:
Fixed and added unit tests to ensure it stays correct.